src/redux/books/books.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable consistent-return */
+
+const ADD_BOOK = 'bookStore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+
+const initialState = [];
+
+export const addBook = (payload) => ({
+  type: ADD_BOOK,
+  payload,
+});
+
+export const removeBook = (payload) => ({
+  type: REMOVE_BOOK,
+  payload,
+});
+
+const reducer = (state = initialState, action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [...state, action.payload];
+
+    case REMOVE_BOOK:
+      return state.filter((book) => book.id !== action.payload);
```

Comment:
- [ ] As pointed out by the previous reviewer, please let's make sure that adding and removing a book actually update the Redux Store. Books added disappear when the page is refreshed.